content/2019/BSDCan/mehnert-MirageOS.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: slides
+title: "MirageOS: minimizing the attack surface and vectors of network services"
+date: 2019-05-18
+author: Hannes Mehnert
+email: hannesm@mastodon.social
+venue: BSDCan 2019
```

Comment:
This one seems not a directly FreeBSD talk. I don't mind to include this on papers.freebsd.org, but I think it's better to confirm the author's opinion first.

Reply:
i was on the fence before your comment.  i'll just remove it.   thanks!